dlang.org.ddoc

```diff
@@ -242,16 +242,16 @@ $(SUBMENU
 )
 NAVIGATION_RESOURCES=
 $(SUBMENU
+    $(ROOT_DIR)acknowledgements.html, Acknowledgments,
+    https://wiki.dlang.org/Books, Books,
     $(ROOT_DIR)tools.html, D-Specific Tools,
```

Comment:
I would just write Tools, we don't write "Books about D" or "Tutorials about D" either